components/Menu/Menu.js

```diff
@@ -18,6 +18,9 @@ export default function Menu({
   const [isTrainingCollections, setIsTrainingCollections] = useState(false);
   const [isTrainingCollectionsClosing, setIsTrainingCollectionsClosing] =
     useState(false);
+  const [isGamingCollections, setIsGamingCollections] = useState(false);
+  const [isGamingCollectionsClosing, setIsGamingCollectionsClosing] =
+    useState(false);
 
```

Comment:
Yes, thank you, we have actually already planned that. We planned the refactoring US because we were thinking about the menu, among other things - it was too much for me to do this on the side.

Reply:
@P-Oliomonte Have a look :blush: